gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/DynamicRoutingConfigProperties.java

```diff
@@ -0,0 +1,20 @@
+package com.netflix.spinnaker.gate.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+
+@Data
+@ConfigurationProperties(prefix = "dynamic-routing")
+public class DynamicRoutingConfigProperties {
+  public static final String ENABLED_PROPERTY = "dynamic-routing.enabled";
```

Comment:
i think for dynamic config this needs to be without a `-` no?
Also, i think `.isEnabled()` will append `.enabled` - I am not 100% sure but that's the pattern I was sticking to and it worked for me

Reply:
The `DynamicConfigService` is backed by the Spring environment. This should be translated by Spring appropriately.